Resources/input/fsh/ISiKCapabilityStatementBasisServer.fsh

```diff
@@ -6,1152 +6,1451 @@ Usage: #definition
-* rest.resource[=].searchRevInclude[+] = "Condition:subject"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "Procedure:patient"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "Procedure:subject"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "Encounter:patient"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "Encounter:subject"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "Coverage:beneficiary"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "Coverage:subscriber"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "Coverage:patient"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "Coverage:payor"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "Account:subject"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "Account:patient"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "AllergyIntolerance:patient"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
-* rest.resource[=].searchRevInclude[+] = "Observation:patient"
-* rest.resource[=].searchRevInclude[=].extension.url = $capabilitystatement-expectation
-* rest.resource[=].searchRevInclude[=].extension.valueCode = #SHALL
+* rest
+  * mode = #server
+  * resource[+]
+    * type = #Patient
```

Comment:
issue: Mir fehlt hier ein motivierender Satz, warum dieses Profil spezifiziert wird in ISiK. Das gilt für alle Profile in diesem CpS. Auch für "triviale" Entscheidugen SOLL zumindest 1 motivierender Satz im Code vorgehlaten werden

suggestion: Im Sinne der DOkumentation reicht eine solche Motivation als Comment (d.h. muss nicht zwingend in der Simplifier Tabelle gerendert werden. Siehe dazu auch gematik OSPO guidelines : https://wiki.gematik.de/display/OSPO/FHIR+Policy#FHIRPolicy-DocumentationofDesignDecisions 

examples:
- Für Patient: "Eine Patientinstanz muss für so gut wie jeden Versorgungsprozess abrufbar sein."
- "